packages/e2e/features/ui/components/storage/storage-browser/action-menu.feature

```diff
@@ -2,12 +2,12 @@ Feature: Create folder with Storage Browser
 
   Background:
     Given I'm running the example "ui/components/storage/storage-browser/default-auth"
+    And I type my "email" with status "CONFIRMED"
+    And I type my password
+    And I click the "Sign in" button
```

Comment:
Nit: would it make sense to add the line `And I click the first button containing "public"`? It's also repeated at the start of every scenario, but I could see it going either way as to whether it could be considered "putting the system in a known state" as Givens are supposed to be used.